vNext/channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -365,9 +365,15 @@ export class PageViewEnvelopeCreator extends EnvelopeCreator {
         }
 
         let bd = telemetryItem.baseData as IPageViewTelemetryInternal;
+
+         // special case: pageview.id is grabbed from current operation id. Analytics plugin is decoupled from properties plugin, so this is done here instead. This can be made a default telemetry intializer instead if needed to be decoupled from channel
+        let currentContextId;
+        if (telemetryItem.ext && telemetryItem.ext.trace && telemetryItem.ext.trace.traceID) {
```

Comment:
What about PageViewPerformance is not a problem also in there?

Reply:
Is not a field in V1 SDK, so it relies on operation.id instead.
https://github.com/microsoft/ApplicationInsights-JS/blob/a96b1a0e77a43e6ba64369cae635c678e009f4f8/JavaScript/JavaScriptSDK/Telemetry/PageViewPerformance.ts#L19-L31